src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -133,7 +173,17 @@ interface ILightningService {
   }: {
     pubkey?: Pubkey
     paymentHash: PaymentHash
-  }): Promise<LnPaymentLookup | LightningServiceError>
+  }): Promise<LnPaymentLookup | LnFailedPartialPaymentLookup | LightningServiceError>
+
+  listSettledPayments({
```

Comment:
I'm wondering if writing this as: ` listSettledPayments(args: ListLnPaymentsArgs): Promise<ListLnPaymentsResult | LightningServiceError>` would not be equivalent, and less verbose?

Reply:
basically not sure the destructuring is very useful for the .types.d.ts. (have not tested whether it's changing something)